crates/rolldown/src/ecmascript/ecma_module_factory.rs

```diff
@@ -201,6 +201,7 @@ impl ModuleFactory for EcmaModuleFactory {
       side_effects,
       module_type: ctx.module_type.clone(),
       has_eval,
+      css_view: None,
```

Comment:
These `Ecma` prefix would be changed later. For now we just leave it untouched and see how it's going.